packages/web/src/components/GameBody.tsx

```diff
@@ -1,41 +1,14 @@
 import React, { useState, useRef, useEffect } from "react";
-import Boot from "game/scenes/Boot";
-import GameScene from "game/scenes/Game";
-import Preload from "game/scenes/Preload";
 import { IonPhaser } from '@ion-phaser/react';
 import DesktopWindow from "components/DesktopWindow";
 import Phaser from "phaser";
 import { useGame } from "hooks/useGame";
-import { getAccountPath } from "@ethersproject/hdnode";
+import { gameConfig } from "game/constants/GameConfig";
 
 export default function GameBody(props: {gameConfig?: Phaser.Types.Core.GameConfig}) {
     const gameRef = useRef<HTMLDivElement>(null);
     
-    const game = useGame({
-        title: "proto",
-        type: Phaser.AUTO,
-        parent: "gameElement",
-        dom: {
-            createContainer: true,
-        },
-        scale: {
-            width: "100%",
-            height: "100%",
-            mode: Phaser.Scale.FIT,
-            fullscreenTarget: "gameElement"
-        },
-        physics: {
-            default: 'matter',
-            matter: {
-                debug: true,
-                gravity: { y: 0 }
-            }
-        },
-        render: {
-            pixelArt: true,
-        },
-        scene: [Boot, Preload, GameScene]
-    }, gameRef);
+    const game = useGame(gameConfig, gameRef);
```

Comment:
nit: gameConfig can prob be an optional arg and get set to the default config inside `useGame`

Reply:
Yep, that was what I intended to do but seems like I forgot to use the optional gameConfig field in the props of the component